src/main/java/macrobase/analysis/BatchAnalyzer.java

```diff
@@ -63,87 +60,19 @@ public AnalysisResult analyze() throws SQLException, IOException, ConfigurationE
     	if(contextualEnabled){
     		contextualAnalyze();	
```

Comment:
Is this only to print, or is the output of this used further down the pipeline (I don't know how context works)?